lib/Doctrine/ORM/Mapping/Driver/AnnotationDriver.php

```diff
@@ -342,7 +342,7 @@ public function loadMetadataForClass($className, ClassMetadata $metadata)
                             'initialValue' => $seqGeneratorAnnot->initialValue
                         ]
                     );
-                } else if ($this->reader->getPropertyAnnotation($property, 'Doctrine\ORM\Mapping\TableGenerator')) {
+                } elseif ($this->reader->getPropertyAnnotation($property, 'Doctrine\ORM\Id\TableGenerator')) {
```

Comment:
I had to change this because of annotations that @lcobucci added in https://github.com/doctrine/annotations/commit/e41281dbaf014b10093e46bc80a53794a4001e6f

I believe we might have fixed a bug thanks to that :)

Not a big deal anyway given what's inside the if I think.